src/overlays/actors/ovl_En_Lift_Nuts/z_en_lift_nuts.h

```diff
@@ -7,11 +7,25 @@ struct EnLiftNuts;
 
 typedef void (*EnLiftNutsActionFunc)(struct EnLiftNuts*, PlayState*);
 
+#define ENLIFTNUTS_GET_FF00(thisx) (((thisx)->params & 0xFF00) >> 8)
+
 typedef struct EnLiftNuts {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x90];
-    /* 0x01D4 */ EnLiftNutsActionFunc actionFunc;
-    /* 0x01D8 */ char unk_1D8[0x180];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ ColliderCylinder collider;
+    /* 0x190 */ SkelAnime skelAnime;
+    /* 0x1D4 */ EnLiftNutsActionFunc actionFunc;
+    /* 0x1D8 */ Vec3f vec_1D8;
+    /* 0x1E4 */ s32 unk_1E4;
+    /* 0x1E8 */ s32 unk_1E8;
+    /* 0x1EC */ s16* ptr_1EC;
+    /* 0x1F0 */ Vec3s jointTable[28];
+    /* 0x298 */ Vec3s morphTable[28];
```

Comment:
```suggestion
    /* 0x1F0 */ Vec3s jointTable[OBJECT_DNT_LIMB_MAX];
    /* 0x298 */ Vec3s morphTable[OBJECT_DNT_LIMB_MAX];
```
You'll need to include `assets/objects/object_dnt/object_dnt.h`